source/manual/data-gov-uk-architecture.html.md

```diff
@@ -0,0 +1,126 @@
+---
+owner_slack: "#datagovuk-tech"
+title: Architectural overview of data.gov.uk
+section: data.gov.uk
+layout: manual_layout
+parent: "/manual.html"
+last_reviewed_on: 2018-04-30
+review_in: 3 months
+---
+![](/manual/images/data-gov-uk-architecture.jpg)
+[paas]: https://docs.cloud.service.gov.uk/#technical-documentation-for-gov-uk-paas
+[publish]: apps/datagovuk_publish
+[find]: apps/datagovuk_find
+[infrastructure]: https://github.com/alphagov/datagovuk_infrastructure
+
+Most of the service is hosted on [GOV.UK PaaS][paas], which divides components in applications (eg Rails apps) and services (databases, messaging services, etc). All applications and services are controlled through cloudfoundry, as used by [GOV.UK PaaS][paas]. Some familiarity with that documentation will be useful to read this document.
```

Comment:
Perhaps we should say "this documentation" or "this manual" instead of "this document" since the medium has changed from a Google Doc?